configs/nodes.yaml

```diff
@@ -73,4 +73,5 @@ nodes:
   comfyui-stream-pack:
     name: "ComfyUI Stream Pack"
     url: "https://github.com/livepeer/ComfyUI-Stream-Pack"
+    branch: "expose_feature_bank_to_comfyui"
```

Comment:
@eliteprox Small request: Next time, can we update the PR title if we include multiple changes in it? In thus case face mesh and feature bank. This will help keep the commit history clean and organised. Thanks!